classes/PrestaShopAutoload.php

```diff
@@ -81,14 +81,24 @@ public static function getInstance()
         return static::$instance;
     }
 
+    /**
+     * Get cache directory.
+     *
+     * @return string
+     */
+    protected static function getCacheDir()
+    {
+        return dirname(__DIR__) . '/cache';
```

Comment:
Same here cache dir is supposed to be `var/cache/dev` (for dev environment) Here this seems to target `classes/cache`

Reply:
![image](https://user-images.githubusercontent.com/2228672/161578860-062abd8f-8354-4d37-a154-c053d9d2a015.png)

yes, it is correct that for dev environment the `_PS_CACHE_DIR_` was different, but with https://github.com/PrestaShop/PrestaShop/pull/24864/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R174 it is impossible to detect the dev environment easily, as this file is loaded before any class is usable, so `_PS_CACHE_DIR_` constant set in:

![image](https://user-images.githubusercontent.com/2228672/161580135-0c727682-b389-413c-999a-8d48a91cae54.png)

cannot be used

is it ok to have one cache location for prod & dev for autoloader?